torchvision/io/__init__.py

```diff
@@ -39,10 +107,11 @@
     "_read_video_meta_data",
     "VideoMetaData",
     "Timebase",
-    'read_image',
-    'decode_image',
-    'encode_jpeg',
-    'write_jpeg',
-    'encode_png',
-    'write_png',
+    "read_image",
+    "decode_image",
+    "encode_jpeg",
+    "write_jpeg",
+    "encode_png",
+    "write_png",
+    "Video",
```

Comment:
For a next PR: reconsider if `Video` is the name we want for the class, or if we want to expose it as something else, like `VideoContainer` or `VideoReader` or something else.